pkgs/by-name/jj/jjui/package.nix

```diff
@@ -6,26 +6,23 @@
 }:
 buildGoModule rec {
   pname = "jjui";
-  version = "0.4";
+  version = "0.5";
 
   src = fetchFromGitHub {
     owner = "idursun";
     repo = "jjui";
     tag = "v${version}";
-    hash = "sha256-PAquHh3VVAhc4Tw1XkyKmwFbpLVkDRCMT+FGtqiydCA=";
+    hash = "sha256-+1KVKevY7aWkAbbHQi06whh3keibdSVEykYDItSMi4I=";
   };
 
   vendorHash = "sha256-MxTwe0S2wvkIy8VJl1p8utTX98zfcwpNgCdnpFAMxO0=";
 
-  postFixup = ''
-    mv $out/bin/cmd $out/bin/jjui
-  '';
-
   passthru.updateScript = nix-update-script { };
 
   meta = {
     description = "A TUI for Jujutsu VCS";
     homepage = "https://github.com/idursun/jjui";
+    changelog = "https://github.com/<repo>/<package>/releases/tag/v${version}";
```

Comment:
```suggestion
    changelog = "https://github.com/idursun/jjui/releases/tag/v${version}";
```

Reply:
Hah. Thanks for catching this. Copy-paste mistake from my notes. Fixed.